wiki/Tournaments/ATT/2021/en.md

```diff
@@ -0,0 +1,143 @@
+| Unseeded | ![][flag_SG] [dantoh000](https://osu.ppy.sh/users/11414361), ![][flag_SG] [\_gt](https://osu.ppy.sh/users/8301957), ![][flag_TH] [KappuChinooo](https://osu.ppy.sh/users/9582525), ![][flag_MY] [TaikOseania](https://osu.ppy.sh/users/14036825) |
+
+## Groups
+
+| Group | Top seed | High seed | Mid seed | Low seed | Unseeded |
+| :-: | :-- | :-- | :-- | :-- | :-- |
+| **A** | ![][flag_MY] [Jerry](https://osu.ppy.sh/users/605973) | ![][flag_PH] [Aiery](https://osu.ppy.sh/users/10363380) | ![][flag_MY] [CrabCow](https://osu.ppy.sh/users/9755504) | ![][flag_SG] [kuudere-desu](https://osu.ppy.sh/users/10694200) | ![][flag_TH] [KappuChinooo](https://osu.ppy.sh/users/9582525) |
+| **B** | ![][flag_ID] [apaajaboleh10](https://osu.ppy.sh/users/5151647) | ![][flag_AU] [Tsubasa2](https://osu.ppy.sh/users/6835183) | ![][flag_PH] [CoEqual](https://osu.ppy.sh/users/14167247) | ![][flag_PH] [Astrid Park](https://osu.ppy.sh/users/12320451) | - |
+| **C** | ![][flag_AU] [r1chyy](https://osu.ppy.sh/users/11499467) | ![][flag_ID] [Katdon\_donKat](https://osu.ppy.sh/users/8089664) | ![][flag_PH] [Kotaro](https://osu.ppy.sh/users/796530) | ![][flag_NZ] [Blujae](https://osu.ppy.sh/users/10613885) | - |
+| **D** | ![][flag_ID] [Volta](https://osu.ppy.sh/users/4154071) | ![][flag_PH] [ezeenS](https://osu.ppy.sh/users/3679969) | ![][flag_PH] [DescriptiCringe](https://osu.ppy.sh/users/10882115) | ![][flag_NZ] [Bwhaa](https://osu.ppy.sh/users/7960435) | - |
+| **E** | ![][flag_TH] [Ponamis](https://osu.ppy.sh/users/7897892) | ![][flag_SG] [Element118](https://osu.ppy.sh/users/2283992) | ![][flag_AU] [AmateurMonkeyYT](https://osu.ppy.sh/users/8379046) | ![][flag_MY] [vun](https://osu.ppy.sh/users/6932501) | ![][flag_SG] [dantoh000](https://osu.ppy.sh/users/11414361) |
+| **F** | ![][flag_ID] [Naverlyn](https://osu.ppy.sh/users/14324722) | ![][flag_SG] [tzechi](https://osu.ppy.sh/users/4662795) | ![][flag_TH] [ConieSan](https://osu.ppy.sh/users/2035344) | ![][flag_PH] [qish](https://osu.ppy.sh/users/17050115) | ![][flag_MY] [TaikOseania](https://osu.ppy.sh/users/14036825) |
+| **G** | ![][flag_ID] [misaki_tk](https://osu.ppy.sh/users/3866964) | ![][flag_SG] [Prehistoria](https://osu.ppy.sh/users/8364237) | ![][flag_AU] [Beat43210](https://osu.ppy.sh/users/5664171) | ![][flag_NZ] [arley\_](https://osu.ppy.sh/users/9039824) | ![][flag_SG] [\_gt](https://osu.ppy.sh/users/8301957) |
+| **H** | ![][flag_PH] [jmeh07](https://osu.ppy.sh/users/2852269) | ![][flag_SG] [Bobfree](https://osu.ppy.sh/users/10096125) | ![][flag_MY] [Bedwyr Aorta](https://osu.ppy.sh/users/10875855) | ![][flag_MY] [nikolpai\_15](https://osu.ppy.sh/users/11584592) | - |
+
+## Mappools
+
+### Group Stage
+
+**[Download the mappack here! (53 MB)](https://drive.google.com/file/d/1_9FfeVHDMf3fMae8ZxMMWkrEESVETm-r/view?usp=sharing)**
```

Comment:
**Not sure if this is a convention in tournament articles**, but exclamation points and using "here" in link title text is discouraged in neutral tone articles. If it is a convention, promptly ignore

Reply:
Lowkey following Official World Cups, but if we're about to change it, maybe something like
> Download the mappack through the attached link

(?)

Reply:
i'd say just leave this as is to follow the world cup standards ya